python/vowpalwabbit/pyvw.py

```diff
@@ -2,11 +2,13 @@
 """Python binding for pylibvw class"""
 
 from __future__ import division
-import pylibvw
+from pylibvw import vw as pylibvw_vw, example as pylibvw_example
```

Comment:
This is a breaking change - any script that uses `pyvw.pylibvw.<>` will no longer work, and since this is the only way to get at label and prediction types right now we cannot change this.